src/app/public/modules/code-block/code-block.component.html

```diff
@@ -0,0 +1,25 @@
+<div class="stache-code-block">
+  <div class="code-block-header">
+    <div class="sky-deemphasized">
+      {{displayName}}
+    </div>
+    <p>
+
+    </p>
+    <button class="copy-to-clipboard-place-holder">Copy</button>
+    <!-- <stache-copy-to-clipboard
```

Comment:
Why is this commented?

Reply:
Because the clip board library is not created yet, that's next.  Once it is in, I will be pulling in that library to use here, and simply uncomment that button.  Currently I am using this PR as a means of testing my travis integration :)